todo/.gitignore

```diff
@@ -1,2 +0,0 @@
-node_modules/
-.DS_Store
```

Comment:
why is this being removed?

Reply:
When I made my .gitignore I made two copies by mistake, one within the todo file and one outside of the todo file. This removed file is the .gitignore that is within the todo file.